Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -264,11 +264,11 @@ public RaycastDirectionMode CurrentRaycastDirectionMode
         }
 
         [SerializeField]
-        [Tooltip("Orientation mode. None = no orienting, Vertical = Face head, but always oriented up/down, Full = Aligned to surface normal completely")]
-        private OrientationMode orientationMode = OrientationMode.Vertical;
+        [Tooltip("How solver will orient model. None = no orieting, TrackedTarget = Face tracked target transform, SurfaceNormal = Aligned to surface normal completely, Blended = blend between tracked transform and surface orientation")]
```

Comment:
>orieting [](start = 58, length = 8)

Nit: orienting

Reply:
err orientation

---
In reply to: [311316917](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5532#discussion_r311316917) [](ancestors = 311316917)

Reply:
```suggestion
        [Tooltip("How solver will orient model. None = no orienting, TrackedTarget = Face tracked target transform, SurfaceNormal = Aligned to surface normal completely, Blended = blend between tracked transform and surface orientation")]
```